packages/app/src/components/Visualization/StartScreen.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import i18n from '@dhis2/d2-i18n'
+import styles from './styles/StartScreen.style'
+import { sGetLoadError } from '../../reducers/loader'
+import PropTypes from 'prop-types'
+import chartErrorImg from '../../assets/chart-error-graphic.png'
+import { apiFetchFavorites } from '../../api/favorites'
+import history from '../../modules/history'
+import { withStyles } from '@material-ui/core/styles'
+
+export class StartScreen extends Component {
+    state = {
+        favorites: [],
+    }
+
+    componentDidMount() {
+        this.getFavorites()
+    }
+
+    getFavorites = async () => {
+        const result = await apiFetchFavorites()
+        this.setState({ favorites: result })
+    }
+
+    getContent = () =>
+        this.props.error ? (
+            <div>
+                <img src={chartErrorImg} alt={i18n.t('Chart error')} />
```

Comment:
Change to a more generic `Visualization error` since we have pivot tables types as well.